src/app/problems/problem-create/problem-create.component.ts

```diff
@@ -1,11 +1,13 @@
 import {Component, OnInit} from '@angular/core';
 import {ActivatedRoute} from '@angular/router';
 import {Subscription} from 'rxjs';
+import {QuestionService} from "@app/problems/_services/question.service";
 
 @Component({
     selector: 'app-problem-create',
     templateUrl: './problem-create.component.html',
-    styleUrls: ['./problem-create.component.scss']
+    styleUrls: ['./problem-create.component.scss'],
+    providers: [QuestionService]
```

Comment:
WHy do we have the provider here?

Reply:
The example I viewed for keeping a service within a module showcased it being done in this manner.